devtools/lightning-inspect.es5.js

```diff
@@ -0,0 +1,825 @@
+(function (global, factory) {
```

Comment:
Build results should not be version controlled. Instead, I can update npm package with es5 bundles.

Reply:
@g-zachar the dist folder (and automatic build step upon each commit with husky) is added intentionally to make sure the built versions are available even if you npm install straight from GitHub

Reply:
That's true, but regular developer should use npm registry package in the first place. Install from github should be only in special cases e.g. in your case, where you are actively contributing, but in this case having to rebuild bundles each time shouldn't be that much of a problem.